linera-views/src/batch.rs

```diff
@@ -148,6 +148,16 @@ impl UnorderedBatch {
         self.key_prefix_deletions = key_prefix_deletions;
         Ok(())
     }
+
+    /// The total number of entries of the batch.
+    pub fn len(&self) -> usize {
+        self.key_prefix_deletions.len() + self.simple_unordered_batch.len()
+    }
+
+    /// Tests whether the batch is empty or not
+    pub fn is_empty(&self) -> bool {
+        self.key_prefix_deletions.len() == 0 && self.simple_unordered_batch.len() == 0
```

Comment:
(I'm surprised this doesn't trigger Clippy's `len_zero` lint.)

```suggestion
        self.key_prefix_deletions.is_empty() && self.simple_unordered_batch.is_empty()
```

Reply:
Ok.